AppImageBuilder.yml

```diff
@@ -0,0 +1,82 @@
+version: 1
+
+AppDir:
+  path: ./AppDir
+  app_info:
+    id: performous
+    name: performous
+    icon: performous
+    version: 1.x
+    exec: usr/bin/performous
+    exec_args: $@
+  apt:
+    arch: amd64
+    allow_unauthenticated: true
+    sources:
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal main restricted universe multiverse
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ focal-updates main restricted universe multiverse
+    include:
+      ## All of these dependencies can be found by downloading the Performous
+      ## package for Ubuntu 20.04 and doing a dpkg -I on it
+      - libaubio5
+      - libavcodec58
+      - libavformat58
+      - libavutil56
+      - libboost-iostreams1.71.0
+      - libboost-locale1.71.0
+      - libboost-program-options1.71.0
+      - libc6
+      - libcairo2
+      - libcpprest2.10
+      - libepoxy0
+      - libfontconfig1
+      - libgcc-s1
+      - libglib2.0-0
+      - libglibmm-2.4-1v5
+      - libicu66
+      - libjpeg8
+      - libopencv-core4.2
+      - libopencv-videoio4.2
+      - libpango-1.0-0
+      - libpangocairo-1.0-0
+      - libpng16-16
+      - libportaudio2
+      - libportmidi0
+      - librsvg2-2
+      - libsdl2-2.0-0
+      - libssl1.1
+      - libstdc++6
+      - libswresample3
+      - libswscale5
+      - libxml++2.6-2v5
```

Comment:
This breaks portability with other systems than ubuntu 20.04 i think

Reply:
It shouldn't. Those dependencies are being installed inside of the AppImage

Reply:
It's yaml, and each operation is executed in the bash shell. So referencing the version should work :)